operator/v1alpha1/types_image_content_source_policy.go

```diff
@@ -65,3 +69,25 @@ type RepositoryDigestMirrors struct {
 	// +optional
 	Mirrors []string `json:"mirrors"`
 }
+
+// RepositoryMirrors holds cluster-wide information about how to handle mirrors in the registries config.
+// Note: this is different from the RepositoryDigestMirrors that the mirrors only
+// work when pulling the images that are referenced by their digests.
+type RepositoryMirrors struct {
+	// source is the repository that users refer to, e.g. in image pull specifications.
+	// +required
+	Source string `json:"source"`
+	// If true, the mirrors only work when pulling the images that are referenced by their digests.
+	// Pulling images by tag can potentially yield different images, depending on which endpoint
+	// we pull from. Forcing digest-pulls for mirrors avoids that issue.
+	// +optional
+	AllowMirrorByTags bool `json:"allowMirrorByTags"`
```

Comment:
omitempty as optional bool.

Reply:
To keep the same implementation as `RepositoryDigestMirrors` we currently have, I tend to not have the `omitempty` to avoid possible null pointer errors.

Reply:
You can't have a null pointer error with a `bool` type; it would need to be `*bool`.  `omitempty` seems completely orthogonal.  Can re-open this thread until this is `omitempty`?  Or do you have another reason for keeping explicit `false` in the serialized content?